core/peerset/src/test/mod.rs

```diff
@@ -225,8 +225,6 @@ fn test_random_api_use() {
                         match (drop_position, incoming_position) {
                             (Some(drop), Some(incoming)) => {
                                 assert!(drop < incoming);
-                                println!("Last messages: {:?}", last_messages);
-                                assert!(last_messages.contains(&Message::Connect(peer_id.clone())));
```

Comment:
Since the `Connect` message has already been removed from the `VecDeque` above, this was a faulty assertion...